main.js

```diff
@@ -0,0 +1,65 @@
+import {
+  validEmail,
+  validPassword,
+  validNickname,
+  validConfirmPassword,
+} from "./utils/valid.js";
+
+const authForm = document.querySelector(".auth_form");
+const authInputs = document.querySelectorAll(".auth_input");
+const password = document.getElementById("login-password");
+const confirmPassword = document.getElementById("login-confirm_password");
+const submitButton = document.querySelector(".submit-button");
+
+authForm.addEventListener("focusout", focusoutAuthInput);
+authForm.addEventListener("keyup", keyupConfirmPassword);
+authForm.addEventListener("click", setIsViewAboutPassword);
+
+submitButton.addEventListener("click", (e) => {
+  if (e.detail === 0) e.preventDefault();
+  console.log("click");
+});
+
+function setIsViewAboutPassword(e) {
+  const { target } = e;
+  const { parentElement } = target;
+
+  if (!parentElement.classList.contains("view-password")) return;
+  if (target.tagName !== "IMG") return;
+
+  const hiddenImg = parentElement.querySelector(".hidden");
+  console.dir(target);
+  console.dir(hiddenImg);
```

Comment:
```suggestion
  console.dir(target);
  console.dir(hiddenImg);
```
commit 전에 vscode에 전체 검색 기능을 활용해서 commit에 추가되지 않아도 되는 console 코드는 정리할 수 있어서 한번 활용 해 보세요! 단축키는 `shift+command+f` , 왼쪽 패널에 돋보기 (search)기능이에요!

Reply:
넵 맞습니다. console을 제대로 지워야하는데 다음엔 꼭 모두 지우겠습니다